src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -36,6 +36,7 @@ public sealed class DurableOrchestrationContext : DurableOrchestrationContextBas
         private string serializedInput;
         private string serializedOutput;
         private string serializedCustomStatus;
+        private string instanceId;

```

Comment:
Can we remove the `instanceId` field here, remove the `InstanceId` property override below, and make `InstanceId` on the base class virtual instead of abstract?